torch/_dynamo/source.py

```diff
@@ -79,7 +79,7 @@ def reconstruct(self, codegen):
         ]
 
     def name(self):
-        return rename_implicit(f"random_value_{self.random_call_index}")
+        return f"random_value_{self.random_call_index}"
```

Comment:
I guess this was also unhygienic before 🤔  Where does this even get bound

Reply:
```
 if (
            self.is_supported_random()
            and all(k.is_python_constant() for k in args)
            and all(v.is_python_constant() for v in kwargs.values())
        ):
            args = [x.as_python_constant() for x in args]
            kwargs = {k: v.as_python_constant() for k, v in kwargs.items()}
            random_call_index = len(tx.random_calls)
            if random_call_index == 0:
                tx.output.initial_random_state = random.getstate()
            example_value = self.value(*args, **kwargs)
            source = RandomValueSource(random_call_index)
            tx.random_calls.append((self.value, args, kwargs))
            return VariableBuilder(tx, source).wrap_unspecialized_primitive(
                example_value
            )
```

Seems a little sus...